### PR TITLE
Add dependencies for RVS package

### PR DESCRIPTION
**Pull request by jkottiku**
ROCm internal and external (3rd party) dependencies for RVS package.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -137,18 +137,24 @@ set ( RVS_VERSION "1.0.0" )
...
 set(CPACK_PACKAGE_CONTACT "ROCM Validation Suite Support <rocm-validation-suite.support@amd.com>")
-#set(CPACK_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}-${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocblas")
-set(CPACK_RPM_PACKAGE_REQUIRES "rocblas")
+
+# Package dependencies
+if (${RVS_OS_TYPE} STREQUAL "ubuntu")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "hip-runtime-amd, comgr, hsa-rocr, rocblas, rocm-smi-lib, libpci3, libyaml-cpp-dev")
+elseif (${RVS_OS_TYPE} STREQUAL "sles")
+set(CPACK_RPM_PACKAGE_REQUIRES "hip-runtime-amd, comgr, hsa-rocr, rocblas, rocm-smi-lib, libpci3, libyaml-cpp0_6")
```
Comment (manoj-freyr) [inline CMakeLists.txt:153]: yaml-cpp-devel is the package for rhel, centos and sles right? We have it updated on our readme page too

**Review by manoj-freyr — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (jkottiku) [reply CMakeLists.txt:153]: For pre-build RVS package we just need the yaml cpp shared lib, so non-dev version of yaml cpp was used in the distro where possible. But to build RVS source code we need dev version of yaml cpp package.

Comment (manoj-freyr) [reply CMakeLists.txt:153]: okay

**Review by manoj-freyr — APPROVED** [review]
looks good, please test with all OSes before merging

**Comment by jkottiku** [comment]
> looks good, please test with all OSes before merging

Tested on all distros - Ubuntu, RHEL, CentOS, SLES & Mariner.

**Review by frepaul — APPROVED** [review]

**Pull request merged**